eos/models/wilson-coefficients.hh

```diff
@@ -168,23 +168,37 @@ namespace eos
          * These two operators are defined via
          *
          * H^eff = 4 G_F / sqrt(2) V_tb V_ts^* (
-         *             C_L alpha_e / (2 pi) * [sbar gamma^mu P_L b] [nubar gamma_mu P_L nu]
-         *             C_R alpha_e / (2 pi) * [sbar gamma^mu P_R b] [nubar gamma_mu P_L nu]
+         *             C_L alpha_e / (2 pi) * [nubar gamma_mu P_L nu] [sbar gamma^mu P_L b] 
```

Comment:
Please keep the original order of [sb] and [nu nubar] currents. We address the operators as ``sbnunu``, not ``nunusb``.